tests/Custom/DesiredCapabilitiesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Driver\Selenium2Driver;
+use Behat\Mink\Tests\Driver\TestCase;
+
+class desiredCapabilitieTest extends TestCase
```

Comment:
Class name should match filename. Probably you don't have test auto-loading issue because you're on Windows.